setup.py

```diff
@@ -1,8 +1,33 @@
 from setuptools import setup
 
+
+"""
+This code is from the python documentation and is
+designed to read in the version number.
+See: https://packaging.python.org/en/latest/guides/single-sourcing-package-version/
+"""
+import codecs
+import os.path
+
+
+def read(rel_path):
+    here = os.path.abspath(os.path.dirname(__file__))
+    with codecs.open(os.path.join(here, rel_path), "r") as fp:
+        return fp.read()
+
+
+def get_version(rel_path):
+    for line in read(rel_path).splitlines():
+        if line.startswith("__version__"):
+            delim = '"' if '"' in line else "'"
+            return line.split(delim)[1]
+    else:
+        raise RuntimeError("Unable to find version string.")
+
+
 setup(
     name="tobac",
```

Comment:
```suggestion
    name=PACKAGE_NAME,
```

Reply:
a suggestion to define a constant for the package name because "tobac" occurs now 3 times in `setup.py` (see l.30, 50)